spec/models/password_strategies_spec.rb

```diff
@@ -3,6 +3,8 @@
 describe Clearance::User do
   subject do
     class UniquenessValidator < ActiveModel::Validator
+      undef validate
```

Comment:
Oh, I missed that; that's definitely what it is. You're defining this class every time the test runs. You could use `stub_const` from RSpec, which will then clean it up after the test.